packages/colorpickers/src/styled/ColorPicker/StyledAlpha.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { Range } from '@zendeskgarden/react-forms';
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { colorStyles, thumbStyles, trackStyles, trackLowerStyles } from './StyledHue';
+
+const COMPONENT_ID = 'colorpickers.colorpicker_alpha';
+
+/**
+ * 1. Adjust spacing in IE11 to match other browsers
+ * 2. Provides height for the input so that the thumb shadow styles are not cut off in IE11.
+ */
+export const StyledAlpha = styled((Range as unknown) as 'input').attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  position: absolute;
+  z-index: 2;
+  /* stylelint-disable-next-line declaration-no-important */
+  margin-top: 0 !important;
+  border-radius: 0;
+  height: ${props => props.theme.space.base * 3}px;
+
+  &::-webkit-slider-thumb {
+    margin-top: -5px;
+  }
```

Comment:
why is this not being set using `thumbStyles`? If it's a hack, it should be commented.